templates.go

```diff
@@ -135,6 +135,10 @@ func getTemplates() *template.Template {
 			(function() {
 				var inputs = document.getElementsByName('rd');
 				for (var i = 0; i < inputs.length; i++) {
+					var idx = inputs[i].value.indexOf('#');
+					if (idx >= 0) {
+						inputs[i].value = inputs[i].value.substr(0, idx);
+					}
```

Comment:
Would you mind adding a brief comment to outline what happens here.

Reply:
Sure. I added comments, I hope it's now easier to understand!